server/tests/test_api.py

```diff
@@ -182,15 +182,15 @@ def test_update_task(self):
         )
         response = ApiView(request).update_task()
         self.assertTrue(response["task_alive"])
-        self.assertTrue(self.rundb.get_run(self.run_id)["results_stale"])
+        self.assertFalse(self.rundb.get_run(self.run_id)["results_stale"])
```

Comment:
Before `get_results()` was not called in `update_task()` for a num_games test. So the results were stale after the call. Now `get_results()` is called unconditionally (since we need the number of games to decide if the test is finished).